docs/relationship-particles.md

```diff
@@ -0,0 +1,392 @@
+---
+nav_order: 6
+---
+
+# Relationship with systemd "particles"
+
+There is an excellent [vision blog entry](https://0pointer.net/blog/fitting-everything-together.html)
+that puts together a coherent picture for how a systemd (and [uapi-group.org](https://uapi-group.org/))
+oriented Linux based operating system can be put together, and the rationale for doing so.
+
+The "bootc vision" aligns with parts of this, but differs in emphasis and also some important technical details - and some of the emphasis and details have high level ramifications.  Simply stated: related but different.
+
+## System emphasis
+
+The "particle" proposal mentions that the desktop case is most
+interesting; the bootc belief is that servers are equally
+important and interesting.  In practice, this is not a real point
```

Comment:
I've just finished reading the particle paper after this tipped me off to it (so thanks for educating me!).  I think this is an oversimplified take on what the paper is saying.  The key point I took away is that the desktop case is a superset of the complications that arise in server/embedded (maybe your point here is that you disagree with that assertion?).  So if you concentrate on the desktop case, then it follows that you also cover 100% of the server case as well (and then some).

Reply:
> (maybe your point here is that you disagree with that assertion?).

Yes, this.  There are some things that are hard on servers that do not exist on desktops in isolation; particularly "distributed systems" interactions.